department-of-veterans-affairs/caseflow
# Allocation data structure

Connects #5709 

### Description
This PR adds the table for allocations.

### Testing Plan
1. Run the migration
1. In the rails console, run `Allocation.new(schedule_period: SchedulePeriod.first, regional_office: "RO33", allocated_days: 2.5)`
1. Confirm that the allocation is saved correctly
